mlir/include/mlir-c/BuiltinAttributes.h

```diff
@@ -558,6 +558,20 @@ mlirDenseElementsAttrGetRawData(MlirAttribute attr);
 // Resource blob attributes.
 //===----------------------------------------------------------------------===//
 
+MLIR_CAPI_EXPORTED bool
+mlirAttributeIsADenseResourceElements(MlirAttribute attr);
+
+/// Unlike the typed accessors below, constructs the attribute with a raw
+/// data buffer and no type/alignment checking. Use a more strongly typed
+/// accessor if possible. If the deleter is non NULL, then it will be called
```

Comment:
Why not make the typed ones also accept deleter while here?

Reply:
Just want touching what I didn't write in case if people were using those. I felt ok changing this API since I just added it.

Mine is I do a follow-up to change the existing APIs and advertise it accordingly?

Reply:
Works for me.